src/PKSim.UI.Starter/ApplicationStartup.cs

```diff
@@ -54,24 +74,82 @@ private static IContainer initializeForStartup(IShell shell)
          {
+            container.RegisterImplementationOf(shell as BaseShell);
             container.RegisterImplementationOf(shell);
+            container.RegisterImplementationOf(UserLookAndFeel.Default);
             container.AddRegister(x => x.FromType<CoreRegister>());
             container.AddRegister(x => x.FromType<InfrastructureRegister>());
             container.Register<ICreateExpressionProfilePresenter, CreateExpressionProfilePresenter>();
             container.Register<ICreateExpressionProfileView, CreateExpressionProfileView>();
+            container.Register<ICreateIndividualPresenter, CreateIndividualPresenterForMoBi>();
+            container.Register<ICreateIndividualView, CreateIndividualView>();
             container.Register<IExpressionProfileToExpressionProfileDTOMapper, ExpressionProfileToExpressionProfileDTOMapper>();
             container.Register<IMoleculePropertiesMapper, MoleculePropertiesMapper>();
             container.Register<ICoreUserSettings, IPresentationUserSettings, UIStarterUserSettings>(LifeStyle.Singleton);
             container.Register<IOntogenyTask, OntogenyTask>();
             container.Register<IEntityTask, EntityTask>();
             container.Register<IRenameObjectDTOFactory, RenameObjectDTOFactory>();
             container.Register<IHistoryManager, HistoryManager<IExecutionContext>>();
+            container.Register<ISubPresenterItemManager<IIndividualItemPresenter>, SubPresenterItemManager<IIndividualItemPresenter>>();
+            container.Register<IBuildingBlockPropertiesMapper, BuildingBlockPropertiesMapper>();
+            container.Register<IObjectBaseDTOFactory, ObjectBaseDTOFactory>();
+            container.Register<IIndividualSettingsPresenter, IndividualSettingsPresenter>();
+            container.Register<IIndividualSettingsView, IndividualSettingsView>();
+            container.Register<IToolTipCreator, ToolTipCreator>();
+            container.Register<IToolTipPartCreator, ToolTipPartCreator>();
+            container.Register<IReportPartToToolTipPartsMapper, ReportPartToToolTipPartsMapper>();
+            container.Register<IIndividualDefaultValueUpdater, IndividualDefaultValuesUpdater>();
+            container.Register<IIndividualSettingsDTOToOriginDataMapper, IndividualSettingsDTOToOriginDataMapper>();
+            container.Register<IParameterToParameterDTOMapper, ParameterToParameterDTOMapper>();
+            container.Register<ISubPopulationToSubPopulationDTOMapper, SubPopulationToSubPopulationDTOMapper>();
+            container
+               .Register<IParameterValueVersionToCategoryParameterValueVersionDTOMapper,
+                  ParameterValueVersionToCategoryParameterValueVersionDTOMapper>();
+            container.Register<ICalculationMethodToCategoryCalculationMethodDTOMapper, CalculationMethodToCategoryCalculationMethodDTOMapper>();
+            container.Register<IIndividualToIIndividualSettingsDTOMapper, IndividualToIIndividualSettingsDTOMapper>();
```

Comment:
WHAT. ARE WE DOING HERE?

Reply:
What do you mean? Manually registering dependencies for the CreateIndividualPresenter.

Are you asking why the list is long, or asking a question about one line of code?